analyzers/tests/SonarAnalyzer.UnitTest/Helpers/SonarLintXmlReaderTest.cs

```diff
@@ -82,6 +82,22 @@ public void SonarLintXmlReader_PropertiesCSharpTrueVBNetFalse_ExpectedValues()
         sut.AnalyzeGeneratedCode(LanguageNames.VisualBasic).Should().BeFalse();
     }
 
+    [TestMethod]
+    public void SonarLintXmlReader_DuplicatedProperties_DoesNotFail()
+    {
```

Comment:
Throwing is asserted via lambdas.
```suggestion
    public void SonarLintXmlReader_DuplicatedProperties_DoesNotFail() =>
        ((Action)(() => CreateSonarLintXmlReader("ResourceTests\\SonarLintXml\\Duplicated_Properties\\SonarLint.xml"))).Should().NotThrow();
```

Reply:
I strongly prefer `@"asdf\asdf"` over double `\\` - this applies everywhere. The `\\` is cumbersome to work with, copy paste and use in general.